docs/internals/howto-release-django.txt

```diff
@@ -265,19 +325,24 @@ OK, this is the fun part, where we actually push out a release!
    that the commit is a security fix and that an announcement will follow
    (:commit:`example security commit <bf39978a53f117ca02e9a0c78b76664a41a54745>`).
 
-#. For a feature release, remove the ``UNDER DEVELOPMENT`` header at the
-   top of the release notes and add the release date on the next line. For a
-   patch release, remove the ``Expected`` prefix and update the release date,
-   if necessary. Make this change on all branches where the release notes for a
-   particular version are located.
-
 #. Update the version number in ``django/__init__.py`` for the release.
    Please see `notes on setting the VERSION tuple`_ below for details
-   on ``VERSION``.
+   on ``VERSION``. Commit message should be similar to (set commit prefix
+   according to the stable branch you are in):
+
+   .. code-block:: text
 
-#. If this is a pre-release package, update the "Development Status" trove
-   classifier in ``setup.cfg`` to reflect this. Otherwise, make sure the
-   classifier is set to ``Development Status :: 5 - Production/Stable``.
+        [5.0.x] Bumped version for 5.0.6 release.
+
+   #. If this is a pre-release package (`example commit for Alpha release
```

Comment:
I converted this to a subitem since both changes, when present, should be done in a single commit.